XCommas.Net/XCommas.Net/XCommasApi.cs

```diff
@@ -453,6 +453,20 @@ public async Task<XCommasResponse<SmartTradeStep>> AddFundsToSmartTradeAsync(Sma
         }
         #endregion
 
+        #region users
+
+        public XCommasResponse<bool> ChangeUserMode(User mode) => this.ChangeUserModeAsync(mode).Result;
```

Comment:
When the User enum has been renamed to UserMode the signature of these two methods should be changed to `(UserMode userMode)`.